src/community/index.md

```diff
@@ -8,7 +8,7 @@ group: https://groups.google.com/a/dartlang.org
 Track the Dart project, get help, and talk with other Dart developers.
 
 We pledge to maintain an open and welcoming environment.
-For details, see our [code of conduct](/code-of-conduct).
+For details, see our [code of conduct](/community/code-of-conduct).
```

Comment:
**nit:** As you touched it, maybe update for inclusive language?
```suggestion
To learn more about our pledge, check out our [code of conduct](/community/code-of-conduct).
```

Reply:
I think the page doesn't really talk about the pledge, but rather the environment itself. Maybe expectations?

```suggestion
To learn more about our expectations, check out
our [code of conduct](/community/code-of-conduct).
```